comb_spec_searcher/strategies/constructor.py

```diff
@@ -195,25 +213,6 @@ def _build_children_param_map(
             )
         return tuple(map_list)
 
-    @staticmethod
-    def _build_param_map(
-        child_pos_to_parent_pos: Tuple[Tuple[int, ...], ...], num_parent_params: int
-    ) -> ParametersMap:
-        """
-        Build the ParametersMap that will map according to the given child pos to parent
-        pos map given.
-        """
-
-        def param_map(param: Parameters) -> Parameters:
-            new_params = [0 for _ in range(num_parent_params)]
-            for pos, value in enumerate(param):
-                parent_pos = child_pos_to_parent_pos[pos]
-                for p in parent_pos:
-                    new_params[p] += value
```

Comment:
I'm trying to remember why, but the fact this is += rather than only assigning once as the altered version would do, I thought was important.